canonical-web-and-design/jaas.ai
# Remove the search functionality

## Done
Removed all search functionality from the site and added a basic redirect to charmhub search

## QA
- Go to /search/?q=mysql and check it redirect correctly
- Check there is no search in the header
- Check the site over for buttons and links and check nothing errors

## Details
Fixes https://github.com/canonical-web-and-design/app-tribe/issues/599
